maths/lucas_series.rb

```diff
@@ -0,0 +1,31 @@
+# A ruby program for Lucas series
+#
+# The Lucas numbers, commonly denoted L(n) form a sequence,
+# called the Lucas series, such that each number is the sum
+# of the two preceding ones, starting from 2 and 1. That is,
+#
+# L(0) = 2, L(1) = 1
+# L(n) = L(n - 1) + L(n - 2), for n > 1.
+#
+# Given n, calculate L(n).
+# Example: 2 1 3 4 7 11 18...
+# Resource: https://en.wikipedia.org/wiki/Lucas_number
+
+def lucas(number)
+  golden_ratio = (1 + 5**0.5) / 2
+  ((golden_ratio**number).round()).to_i
```

Comment:
We use this formula to solve the Fibonacci problem: https://github.com/TheAlgorithms/Ruby/blob/master/maths/fibonacci.rb.

Reply:
Yeah, we do. Golden ratio is also used to calculate the Lucas series as well. Can you please have a look at this video? 
Video URL: https://youtu.be/PeUbRXnbmms